core/trino-main/src/test/java/io/trino/execution/TestColumnTypeCoercion.java

```diff
@@ -38,7 +38,7 @@
 
 @TestInstance(PER_CLASS)
 @Execution(CONCURRENT)
-public class TestCreateTableColumnTypeCoercion
+public class TestColumnTypeCoercion
```

Comment:
Should we provide some context on ColumnTypeCoercion during creation `TestColumnTypeCoercion` is a bit generic as it could during select as well right ?

Reply:
I renamed to `TestCreateAndAlterTableColumnTypeCoercion`.